packages/google_maps_flutter/google_maps_flutter_web/lib/src/utils.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:web/web.dart';
+
+/// Convenience method to create a new [HTMLDivElement] element.
+HTMLDivElement createDivElement() {
```

Comment:
I didn't find a way to have this utility for both `lib/src` and the example app, while keeping it private to the package. So I kept this copy internal only and added a second version for the example app's integration tests.

Reply:
One thing you can do with this is import from `package:google_maps_flutter_web/lib/src/utils.dart` (a "private" file), which should be allowed from tests (?)